sample/SampleApp/SampleApp.cpp

```diff
@@ -0,0 +1,118 @@
+// ConsoleApplication1.cpp : Defines the entry point for the console application.
+//
+
+#include <windows.h>
+#include <string>
+#include <assert.h>
+#include <playfab_c/PlayFabClientApi_c.h>
+
+PlayFabStateHandle stateHandle{ nullptr };
+PlayFabUserHandle userHandle{ nullptr };
+
+int main()
+{
+    HRESULT hr = PlayFabInitialize(&stateHandle);
+    assert(SUCCEEDED(hr));
+
+    {
+        PlayFabLoginWithCustomIDRequest request{};
+        request.createAccount = false;
+        request.customId = "GettingStartedGuide";
+        request.titleId = "E18D7";

```

Comment:
It seems odd to make the title specify the title id every time they make a sign in request. Could the title id be part of the call to initialize and have it stored in the state handle?

Reply:
@pgilmorepf maybe you could clarify here?  I originally suggested that design, but apparently there is a reason to allow each auth context to specify a different titleId. 

---
In reply to: [596135468](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r596135468) [](ancestors = 596135468)

Reply:
that sounds odd to me since the current PlayFab SDK doesn't do this.
the TitleId is a static setting (or possible a shared setting if you are using the instanced classes) but in both cases, it isn't provided on each API call